### PR TITLE
Upload draft of translated readme

### PR DESCRIPTION
**Pull request by lovit**
- translate readme from `ko` to `en`

### CODE REVIEW

File: README_en.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+Cosine is more appropriate than Euclidean to measure the distance between two documents which are represented with high-dimensional (or sometimes, sparse) vectors.
+However, scikit-learn k-means package only provides Euclidean based k-means not yet.
+In addition, labeling clusters is very helpful to interpret the clustering results.
+
+Spherical k-means works well both with sparse vector representation such as Bag-of-Words model or distributed representation such as Doc2Vec or other document embedding methods.
+In lower dimensional vector space, Silhouette score method is useful to define the number of clusters (`k`).
+However Silhouette score method does not works well anymore in high-dimensional vector space such as Bag-of-Words Models and Doc2Vec.
+One of heuristic methods to define the cluster numbers is to train k-means with large `k` first and then merge similar ones.
+The heuristic method is also useful to prevent Uniform effect which is the phenomenon that the size of all clusters leads similar.
```
Comment (hank110) [inline README_en.md]: ~ in preventing the uniform effect, which causes the size of all clusters to be similar

File: README_en.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+`soyclustering` provides Spherical k-means (k-means which uses Cosine as distance metric) and keyword extraction based clustering labeling function.
+And more, the function visualizing pairwise distances between centroids will help you to check whether redundant clusters exist and merging similar clusters does you to remove the redundant ones.
```
Comment (hank110) [inline README_en.md]: ~ and allows redundant clusters to be merged into a same cluster.

File: README_en.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+Also `soyclustering` provide fast initialization method which is proper to high-dimensional vector space.
```
Comment (hank110) [inline README_en.md]: Additionally, `soyclustering` provides a fast initialization method that is effective for clustering in a high-dimensional vector space.

Comment (lovit) [reply README_en.md]: all conversations are resolved by pr #8

**Pull request merged**